### DISCUSSION TITLE
Support QR codes to Add Device

### DISCUSSION THREAD

**Discussion by x-N0** [spacedriveapp/spacedrive#840] [Feature Request]
![image](https://github.com/spacedriveapp/spacedrive/assets/51316342/b21f63bc-c06d-4f4a-9e65-b06e93d99de9)

I think by itself the title and the screen-capture are self-explanatory:

Why not add the possibility to scan the device ID as QR code?

The Phone-to-PC integration will be at ease if this is done, and prevents keeping the Device IDs copy and pasted everywhere (I assure u people will try not to type it char by char)

If u thought about this, good job, else give it a try

Keep it up, guys!

**Comment by utkubakir** [comment]
Hi, that dialog is just a placeholder and we obviously plan to make an easy way to connect. This will most likely be a 4-digit pin and a QR code. Thanks!
